README.md

```diff
@@ -23,7 +28,7 @@ Below describes the steps needed to uninstall the plugin.
 Below describe the different commands the plugin offers
 
 ### Artifacts Deploy
-* Used to deploy all the direct and transitive package dependencies of a python project. 
+* Used to deploy all the direct and transitive package dependencies of all downstream python projects. Any directory that includes a `poetry.lock` file is considered to be a python project.  
```

Comment:
Q: What does "downstream" mean here?

Reply:
Meaning it looks for python projects at the current directory and anything below. Could not think of a better term to use.

Reply:
Can a poetry project even contain other poetry projects?

Reply:
Yes it could, though I dont see many reasons why you would do this?